controller-server/src/main/java/com/yahoo/vespa/hosted/controller/notify/Notifier.java

```diff
@@ -56,7 +57,12 @@ public void dispatch(List<Notification> notifications, NotificationSource source
 
     private boolean skipSource(NotificationSource source) {
         // Limit sources to production systems only. Dev and test systems cause too much noise at the moment.
-        return source.jobType().map(t -> !t.isProduction()).orElse(false);
+        if (source.zoneId().map(z -> z.environment() != Environment.prod).orElse(false)) {
+            return true;
+        } else if (source.jobType().map(t -> !t.isProduction()).orElse(false)) {
+            return true;
+        }
+        return false;
```

Comment:
Use `!source.isProduction()`?

Reply:
I would like to exclude staging and systemtest for now. Too many emails in the beginning will just be annoying.